types/bootstrap4-toggle/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for bootstrap4-toggle 3.6
+// Project: https://github.com/gitbrent/bootstrap4-toggle, https://gitbrent.github.io/bootstrap4-toggle/
+// Definitions by: Mitchell Grice <https://github.com/gricey432>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
```suggestion
// TypeScript Version: 2.7
```
recent jQuery depends on this in DT:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/index.d.ts#L27

Reply:
thx

Where can I get more info about what TypeScript Version should be set?

Reply:
> Where can I get more info about what TypeScript Version should be set?

@adaamz that really depends, this minimum DT version comes from this feature:

https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-33-or-above

but in case of this DT package, the minimum DT version depends on referenced DT type.
So it is jQuery types detail that mandates what is the minimal version used here.
Not easy to xplain in simple terms in the README IMO, unless we ask a magician: @jablko

Reply:
@peterblazejewicz @adaamz Hi! My understanding is if the Minimum TypeScript Version is less than the lowest supported version, [it's effectively ignored](https://github.com/microsoft/DefinitelyTyped-tools/blob/0ee296ac4e12f9b23be82cd6de82e833eb68d053/packages/definitions-parser/src/packages.ts#L536) and can be removed?
```Shell
npm info @types/bootstrap4-toggle typeScriptVersion
3.3
```

The current lowest supported version [is 3.3](https://github.com/microsoft/DefinitelyTyped-tools/blob/0ee296ac4e12f9b23be82cd6de82e833eb68d053/packages/typescript-versions/src/index.ts#L73). You're free to support older versions, they're just not verified by the infrastructure ([roughly discussed](https://github.com/DefinitelyTyped/DefinitelyTyped#older-versions-of-typescript-31-and-earlier)). If jquery did require a supported version, I'd expect the infrastructure to [check that your package matched](https://github.com/microsoft/DefinitelyTyped-tools/blob/0ee296ac4e12f9b23be82cd6de82e833eb68d053/packages/definitions-parser/src/check-parse-results.ts#L76). I think the Minimum TypeScript Version is basically used to maintain the [tsX.X tags](https://github.com/DefinitelyTyped/DefinitelyTyped#user-content-older-versions-of-typescript-31-and-earlier:~:text=%40types%20packages%20have%20tags%20for%20versions%20of%20TypeScript%20that%20they%20explicitly%20support). And to [select which versions are tested](https://github.com/microsoft/dtslint/blob/952086da5617f3f2c94bc9aad849fe51732eb00e/src/index.ts#L155-L182).